mantle/kola/tests/coretest/core.go

```diff
@@ -38,11 +40,32 @@ var offServices = []string{
 }
 
 func init() {
+
 	register.RegisterTest(&register.Test{
-		Name:        "basic",
-		Description: "Verify basic functionalities like SSH, systemd services, useradd, etc.",
-		Run:         LocalTests,
-		ClusterSize: 1,
+		Name:             "basic.bios",
```

Comment:
Maybe we should keep this one as just `basic` since it's the one test that can run on all arches and platforms?

Reply:
Thank you for that suggestion.